flag.go

```diff
@@ -40,6 +40,14 @@ type FlagLoader struct {
 
 	// Args defines a custom argument list. If nil, os.Args[1:] is used.
 	Args []string
+
+	// UsageFunc an optional function that is called to set a flag.Usage value
+	//  The input is the raw flag name, and the output should be a string
+	//  that will used in passed into the flag for Usage.
+	UsageFunc func(name string) string
```

Comment:
Can you remove the spaces for lines 2 and 3. Also we should add another line documenting the default behavior.